reference/README.md

```diff
@@ -22,172 +22,173 @@ To contribute, please go to the [concepts][concepts], [paradigms][paradigms], [s
-| `type-annotations`                                               | [Swift](../languages/swift/exercises/concept/basics/.docs/instructions.md)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
-| `type-conversion`                                                | [Dart](../languages/dart/exercises/concept/numbers/.docs/instructions.md)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
-| `unicode-identifiers`                                            | [Julia](../languages/julia/exercises/concept/🕑🕤🕗/.docs/instructions.md)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
```

Comment:
This line shouldn't have been changed.

Reply:
This is what prettier does to tables.   If @angelikatyborska hadn't run prettier then it would have failed the formatted check.

The real solution is to just remove this file from changes then regenerate the file.